scripts/tools/extract-tools-github.js

```diff
@@ -1,4 +1,5 @@
 const axios = require('axios');
+require('dotenv').config()
```

Comment:
It's needed because I whenever this script in local, I've to replace the `process.env` with my token there. This file is not part of next js folders so it doesn't take the variables from `.env` or `.env.local` by default.

Reply:
but we do not have .env in website, right?

Reply:
Yeah, this file is part of `.gitignore`, but we need some env variables to run the scripts or run Netlify functions. So, this file shouldn't be part of GitHub repo.

Reply:
> but these are already in Netlify

These files are not part of Netlify or Next js build, so they won't be able to take `.env` or `.env.local` files from the directory for local environment setup. 

We also know that tools workflow file run in local dev setup only, not in Github actions workflow, so to provide it with proper env setup, we can add this line to get the variables from `.env` file. Also, it won't affect any changes or usage inside Github actions as well.